src/regression.js

```diff
@@ -1,41 +1,6 @@
-const DEFAULT_OPTIONS = { order: 2, precision: 2, period: null };
+const { round, deriveDataProperties } = require('./utils');
```

Comment:
CI said to extract common code into utilities.
It also complained about this file being too large.